res/database/Default/user/scr0llbaer.xml

```diff
@@ -358,7 +356,7 @@ Format: Filmed lectures. Laid-back and to lay down.</en>
 				<job index="1" function="4" required="1" />
 				<job index="2" function="8" required="1" />
 				<job index="3" function="128" required="1" />
-				<job index="4" function="2048" gender="2" required="0" />
+				<!--job index="4" function="2048" gender="2" required="0" /--> <!-- TODO: Eventually reactivate, should casting models become possible -->
 				<job index="5" function="64" required="0" />
```

Comment:
For consistency, could you fix the index values.(currentyl 1,2,3,5)

Reply:
ack